scripts/release.sh

```diff
@@ -1,39 +1,51 @@
-#!/bin/bash
+#!/bin/sh
 
-RELEASE_TYPES="('major', 'minor', 'patch')"
+RELEASE_TYPES="('major', 'minor', 'patch ,'automatic_version_bump')"
```

Comment:
isn't there a typo here? missing ' character after patch
why do we need an automatic_version_bump release type?

Reply:
The idea is to be able to run the script manually and automatically. automatic_version_bump is not really a release type, but when selected it proceeds to the automatic release type selection (getting release type from last commit message)